.gitignore

```diff
@@ -118,3 +118,7 @@ env/
 
 ### Configuration files ###
 .env
+
+
+.vagrant 
+gunicorn_access.log
```

Comment:
It could be nice to switch this to `*.log` - that wildcard will then catch the error log too, and any others that we could add later

Reply:
I have updated this